src/modules/item/item.service.ts

```diff
@@ -108,14 +143,16 @@ export default class ItemService {
 
 	private formatItems(items: ItemPrismaProperties[]): ItemWithProperties[] {
 		return items.map((element) => {
-			const { ItemFolder, ItemBlob, ItemDocs, ItemShortcut, ...strippedElement } = element;
+			const { ItemFolder, ItemBlob, ItemDocs, ItemShortcut, ItemStarred, ...strippedElement } =
+				element;
 
 			return {
 				...ItemBlob,
 				...ItemFolder,
 				...ItemDocs,
 				...ItemShortcut,
 				...strippedElement,
+				isStarred: (ItemStarred && ItemStarred.length > 0) ?? false,
```

Comment:
ItemStarred should not be able to be null/undefined

```suggestion
				isStarred: ItemStarred.length > 0,
```